src/ToastNotification/ToastNotificationContainer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace PackagedUi\Fusion\ToastNotification;
+
+use Packaged\Glimpse\Tags\Div;
+use Packaged\Ui\Html\HtmlElement;
+use PackagedUi\BemComponent\BemComponentTrait;
+use PackagedUi\Fusion\Component;
+use PackagedUi\Fusion\ComponentTrait;
+
+class ToastNotificationContainer extends Div implements Component
+{
+  use BemComponentTrait;
+  use ComponentTrait;
+
+  /** @var ToastNotification[] */
+  protected $_toastNotifications = [];
+
+  /** @var int */
+  protected $_delay = 5000;
+
+  /** @var ToastNotificationPosition */
+  protected $_position;
+
+  /**
+   * @inheritDoc
+   */
+  public function getBlockName(): string
+  {
+    return 'toast-notification-container';
+  }
+
+  /**
+   * @param ToastNotification $toastNotification
+   *
+   * @return ToastNotificationContainer
+   */
+  public function addToastNotification(ToastNotification $toastNotification): ToastNotificationContainer
+  {
+    $this->_toastNotifications[] = $toastNotification;
+    return $this;
+  }
+
+  /**
+   * @param int $delay
+   *
+   * @return ToastNotificationContainer
+   */
+  public function setDelay(int $delay): ToastNotificationContainer
+  {
+    $this->_delay = $delay;
+    return $this;
+  }
+
+  /**
+   * @param ToastNotificationPosition $position
+   *
+   * @return ToastNotificationContainer
+   */
+  public function setPosition(ToastNotificationPosition $position): ToastNotificationContainer
+  {
+    $this->_position = $position;
+    return $this;
+  }
+
+  /**
+   * @return ToastNotificationContainer
+   */
+  public function removable(): ToastNotificationContainer
+  {
+    foreach($this->_toastNotifications as $toastNotification)
+    {
+      $toastNotification->removable();
+    }
+    return $this;
+  }
```

Comment:
Lets remove this.    If you add another notification after calling this, it would not be removable.  which is unexpected.